### PR TITLE
feat: added faq section

### PR DESCRIPTION
**Pull request by jayeshxn**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [ ] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [ ] Check the commit's or even all commits' message styles matches our requested structure.
- [ ] Check your code additions will fail neither code linting checks nor unit test.
- [ ] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -85,7 +85,7 @@ typings/
 .yarn-integrity
 
 # dotenv environment variables file
-.env
+.env.development
```
Comment (Shurtu-gal) [inline .gitignore:88]: Duplicate

File: src/components/FAQSection/AccordionItem.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  return (
+    <FAQWrapper>
+      <QuestionContainer onClick={handleToggle} active={active}>
+        <QuestionText>{question}</QuestionText>
+        <PlusIcon active={active} />
+      </QuestionContainer>
+      <AnswerContainer
+        active={active}
+        ref={contentEl}
+        style={active ? { height: contentEl.current.scrollHeight } : { height: '0px' }}
```
Comment (Shurtu-gal) [inline src/components/FAQSection/AccordionItem.jsx]: Try to use transition upon max-height on the container for smoother animation.

**Comment by jayeshxn** [comment]
@Shurtu-gal Please review

File: src/components/FAQSection/PlusIcon.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+const BarOne = styled.div`
+  ${tw`
+  `}
+  background-color: ${(props) => (props.active ? '#91F9D3' : '#D9D9D9')};
+  width: 16px;
+  height: 2px;
+  position: absolute;
+  top: 7px;
+  left: 0;
+  transition: transform 100ms;
```
Comment (Shurtu-gal) [inline src/components/FAQSection/PlusIcon.jsx]: Put a [transition-timing-function](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-timing-function) out here.

**Review by Shurtu-gal — CHANGES_REQUESTED** [review]
@starkjay21 some small changes

**Comment by Shurtu-gal** [comment]
Also run prettier once.

**Comment by jayeshxn** [comment]
@Shurtu-gal I've made the changes. Please check it out.

**Review by Shurtu-gal — APPROVED** [review]
LGTM 🚀

**Pull request merged**